### DISCUSSION TITLE
Does View has to be a Window control?  Can it be a Page control?

### DISCUSSION THREAD

**Discussion by mwdpb** [canton7/Stylet#322] [Q&A] [answered]
I'm refactoring a GUI project that has a shell view with multiple tabs.  Each tab corresponds to a Page control.  I wonder if Stylet view can be a Page control or it has to be a Window control?  If latter, what options do I have in order to use Stylet?  Thanks in advance!

**Comment by canton7** [comment, marked as answer]
Views are normally Windows (which are shown using `.ShowWindow()` or `.ShowDialog()` using the `IWindowManager`), or UserControls (which are shown inside a `ContentControl`. You could probably have a view which is a Page, but I can't think of a situation where that would be useful.

For a TabControl, the recommended way is [documented here](https://github.com/canton7/Stylet/wiki/Screens-and-Conductors#conductortcollectiononeactive). The ShellViewModel owns a collection of child ViewModels (one per tab), and the ShellView binds those to tabs through a templated ContentControl.

**Reply by mwdpb** [reply]
UserControl it is!!  Thank you!
